include/eve/detail/function/simd/x86/slide_left.hpp

```diff
@@ -0,0 +1,147 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/implementation.hpp>
+#include <eve/conditional.hpp>
+
+namespace eve::detail
+{
+  template<real_scalar_value T, typename N, x86_abi ABI, std::ptrdiff_t Shift>
+  EVE_FORCEINLINE wide<T,N,ABI> slide_left_ ( EVE_SUPPORTS(sse2_)
+                                            , wide<T,N,ABI> v, index_t<Shift>
+                                            ) noexcept
```

Comment:
I think here we might be missing aggregated case for 2 registers.
Those can be done fairly well with alignr, similar to what you do for avx.

On the other hand, requirement on zero might be tricky.

Reply:
This can be done mater as a shuffle. Also requirement on zero can be done like with if_else by detecting we pas eve::zero.